src/TodoListItem.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const TodoListItem = (props) => {
+  return (
+    <li> {props.item.title}</li>
+  )
+}
+
+export default TodoListItem;
```

Comment:
You should configure your editor or use an `.editorconfig` file to have consistent newlines at the end of files

Reply:
Is this a plug-in? Does it format like Prettier and ESLint?